app/views/posts/_form.html.erb

```diff
@@ -0,0 +1,22 @@
+<%= form_with(model: post, local: true) do |form| %>
+  <% if post.errors.any? %>
+    <div id="error_explanation">
+      <h2><%= pluralize(post.errors.count, "error") %> prohibited this post from being saved:</h2>
+
+      <ul>
+        <% post.errors.full_messages.each do |message| %>
+          <li><%= message %></li>
+        <% end %>
+      </ul>
+    </div>
+  <% end %>
```

Comment:
- View-related logic should be encapsulated in View helpers `(app/helpers/*_helper.rb)`